src/boot/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// This is the main entry point for the Hypothesis client in the host page
+// and the sidebar application.
+
+/* global __MANIFEST__ */
+
+var boot = require('./boot');
+
+// The HYP* properties are set (if at all) by the service which is serving the
+// Hypothesis client to tell it where to load the sidebar and assets from.
+var appHtmlUrl = window.__HYP_APP_HTML_URL__ || 'https://hypothes.is/app.html';
+var assetRoot = window.__HYP_CLIENT_ASSET_ROOT__ || 'https://hypothes.is/assets/client/';
```

Comment:
How is this going to work with unpkg if the default is `https://hypothes.is/assets/client/`, or is your intent to change that later?